eZ/Bundle/EzPublishCoreBundle/Resources/config/papi.yml

```diff
@@ -53,6 +53,13 @@ services:
         factory_method: buildStorageEngine
         public: false
 
+    ezpublish.spi.search_handler:
+        alias: ezpublish.spi.search_engine
```

Comment:
Well, you set `ezpublish.spi.search_engine` private, and it's SPI.
But that's nitpicking. However, we'll need at some point to clean this up and walk through services and decide which one should be private or not. Note that *private* means that you can't directly get the service from the container, so only by injection.